composer.json

```
@@ -20,8 +20,8 @@
         }
     ],
     "require": {
-        "php": ">=7.2",
-        "symfony/finder": "^5.0"
+        "php": ">=8.0",
+        "symfony/finder": "^6.0"
```

Let's try required `symfony/finder` 5 _or_ 6 here and bumping the required PHP version to `7.4`.

```suggestion
        "php": ">=7.4",
        "symfony/finder": "^5.0 || ^6.0"
```